addons/mail/models/mail_activity_template.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class MailActivityTemplate(models.Model):
+    _name = 'mail.activity.template'
```

Comment:
As it is a sub-model of plan, would maybe name it like 'mail.activity.plan.template' ? or 'mail.activity.type.plan' ?